app/assets/javascripts/report-a-problem.js

```diff
@@ -5,26 +5,140 @@
   var ReportAProblem = function ($container) {
     this.$container = $container;
     var $form = $container.find('form'),
-        form = new GOVUK.ReportAProblemForm($form);
+        form = new GOVUK.ReportAProblemForm($form),
+        renderOriginal = this.renderOriginal.bind(this),
+        renderVariant = this.renderVariant.bind(this);
 
+    $form.on("reportAProblemForm.success", this.showConfirmation.bind(this));
+    $form.on("reportAProblemForm.error", this.showError.bind(this));
+
+    if (ReportAProblem.isBeingTestedOnThisPage()) {
+      this.multivariateTest = new GOVUK.MultivariateTest({
+        name: 'report-a-problem-redesign-ab-test',
+        contentExperimentId: "SnpcHld1SJuQig-_SsaN_Q",
+        cohorts: {
+          variant_0: {variantId: 0, callback: renderOriginal},
+          variant_1: {variantId: 1, callback: renderVariant}
+        }
+      });
+
+      $form.on("reportAProblemForm.success", this.trackSuccessfulFeedback.bind(this));
+    } else {
+      renderOriginal();
+    }
+  };
+
+  ReportAProblem.isBeingTestedOnThisPage = function() {
+    var slugsBeingTested = [
+          '/bank-holidays',
+          '/vehicle-tax-rate-tables',
+          '/contact-the-dvla',
+          '/check-uk-visa',
+          '/life-in-the-uk-test',
+          '/government/publications/application-to-transfer-or-retain-a-vehicle-registration-number',
+          '/government/publications/income-tax-tax-relief-for-expenses-of-employment-p87',
+          '/calculate-state-pension',
+          '/calculate-your-holiday-entitlement',
+          '/am-i-getting-minimum-wage'
+        ],
+        slug = ReportAProblem.getCurrentSlug();
+
+    return slugsBeingTested.indexOf(slug) > -1;
```

Comment:
I don't think that this will work for subpages of guides and smart answers (eg `/check-uk-visa/y/afghanistan`)